Description: Using jquery.jstree with cacti should use jquery and jquery-cookie from
 existing packages.
 .
 This patch replaces embedded paths by system locations of jquery(|.cookie)
Bug-Debian: http://bugs.debian.org/679980
Forwarded: not needed
Author: Paul Gevers <elbrus@debian.org>
Date: Thu, 28 Mar 2013 21:45:43 +0100


--- a/include/top_graph_header.php
+++ b/include/top_graph_header.php
@@ -84,8 +84,8 @@
 	<link href="<?php echo $config['url_path']; ?>include/main.css" type="text/css" rel="stylesheet">
 	<link href="<?php echo $config['url_path']; ?>images/favicon.ico" rel="shortcut icon"/>
 	<script type="text/javascript" src="<?php echo $config['url_path']; ?>include/layout.js"></script>
-	<script type="text/javascript" src="<?php echo $config['url_path']; ?>include/js/jquery/jquery.js" language="javascript"></script>
-	<script type="text/javascript" src="<?php echo $config['url_path']; ?>include/js/jquery/jquery.cookie.js" language="javascript"></script>
+	<script type="text/javascript" src="/javascript/jquery/jquery.js" language="javascript"></script>
+	<script type="text/javascript" src="/javascript/jquery-cookie/jquery.cookie.js" language="javascript"></script>
 	<script type="text/javascript" src="<?php echo $config['url_path']; ?>include/js/jquery/jquery.jstree.js"></script>
 	<script type="text/javascript" src="<?php echo $config['url_path']; ?>include/jscalendar/calendar.js"></script>
 	<script type="text/javascript" src="<?php echo $config['url_path']; ?>include/jscalendar/lang/calendar-en.js"></script>
--- a/lib/graph_export.php
+++ b/lib/graph_export.php
@@ -1718,9 +1718,9 @@
 	copy("$cacti_root_path/images/shadow_gray.gif", "$dir/shadow_gray.gif");
 
 	/* java scripts for the tree */
-	copy("$cacti_root_path/include/js/jquery/jquery.js", "$dir/js/jquery.js");
+	copy("/usr/share/javascript/jquery/jquery.js", "$dir/js/jquery.js");
 	copy("$cacti_root_path/include/js/jquery/jquery.jstree.js", "$dir/js/jquery.jstree.js");
-	copy("$cacti_root_path/include/js/jquery/jquery.cookie.js", "$dir/js/jquery.cookie.js");
+	copy("/usr/share/javascript/jquery-cookie/jquery.cookie.js", "$dir/js/jquery.cookie.js");
 
 	/* theme info for java scripts */
 	copy("$cacti_root_path/include/js/jquery/themes/default/style.css", "$dir/js/style.css");
